dotCMS/src/main/java/com/dotmarketing/business/web/HostWebAPI.java

```diff
@@ -31,6 +32,14 @@
 	
 	public Host getCurrentHost(HttpServletRequest req) throws DotDataException, DotSecurityException, PortalException, SystemException;
 
+	/**
+	 * Return the current host using the current request
+	 * @return
+	 * @throws DotDataException
+	 * @throws DotSecurityException
+	 */
+	public Host getCurrentHost() throws DotDataException, DotSecurityException;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifier 'public' on method 'getCurrentHost': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)